### PR TITLE
Updated Projects.js

### PR DESCRIPTION
**Pull request by HiraIjaz**

### CODE REVIEW

File: Cash Register.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  let Curr = {
+      "PENNY": .01,
+      "NICKEL": .05,
+      "DIME": .10,
+      "QUARTER": .25,
+      "ONE": 1.00,
+      "FIVE": 5.00,
+      "TEN": 10.00,
+      "TWENTY": 20.00,
+      "ONE HUNDRED": 100.00
```
Comment (Touseef-ahmad-arbisoft) [inline Cash Register.js]: change this according to pakistani currency 1, 2, 5, 10, 20, 50, 100, 500, 1000, 5000

File: Palindrome Checker.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function palindrome(str) {
+    str=str.replace(/\s|\W|_/g,"") //removed all the punctuations
+    str=str.toLowerCase() //converted string to lowercase
+    let rstr=str.split("").reverse().join(""); //reversed string
+    return str===rstr; //compared orignal string with reversed string
+  }
+  
+  console.log(palindrome("A man, a plan, a canal. Panama"));
```
Comment (Touseef-ahmad-arbisoft) [inline Palindrome Checker.js]: whats the complexity of this code?

**Comment by Touseef-ahmad-arbisoft** [comment]
use airbnb javascript style guide, https://github.com/airbnb/javascript

File: Caesars Cipher.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+function rot13(str) {
+    let result=""
```
Comment (Touseef-ahmad-arbisoft) [inline Caesars Cipher.js]: add space around = 
`let result = "";`

File: Caesars Cipher.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+function rot13(str) {
+    let result=""
+    
+    for (let i=0; i<str.length; i++){
+      if(str[i].match(/[A-Z]/)){
+        let num=str.charCodeAt(i);
+        if(num-13<65){
+          num=90-((13-(num-65)-1));
```
Comment (Touseef-ahmad-arbisoft) [inline Caesars Cipher.js]: `num = 90 - ((13 - (num - 65) - 1));`

File: Cash Register.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+function totalCash(cid){ //calculates total cash in Cash register
```
Comment (Touseef-ahmad-arbisoft) [inline Cash Register.js]: `function totalCash(cid) {`

File: Cash Register.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+          temp[1] += Curr[elem[0]];
+          elem[1] -= Curr[elem[0]];
+          change -= Curr[elem[0]];
+          change = change.toFixed(2);
+        }
+        if (temp[1] > 0) {
+          changeArray.push(temp);
+        }
+      }
+      console.log(changeArray);
```
Comment (Touseef-ahmad-arbisoft) [inline Cash Register.js]: fix indentation here

**Review by Touseef-ahmad-arbisoft — CHANGES_REQUESTED** [review]

File: Cash Register.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        change = change.toFixed(2);
+      }
+      if (temp[1] > 0) {
+        changeArray.push(temp);
+      }
+    }
+    
+  return {changeArray,change};       
+}
+
```
Comment (Touseef-ahmad-arbisoft) [inline Cash Register.js:30]: const currency = "PKR" // "USD"

File: Palindrome Checker.js (hunk 2 above)
Comment (HiraIjaz) [reply Palindrome Checker.js]: time complexity of .split, .reverse and join is O(n), so it makes it O(3n) which is same as O(n).

**Pull request merged**